django/contrib/admin/templates/admin/filter.html

```diff
@@ -1,8 +1,12 @@
 {% load i18n %}
-<h3>{% blocktranslate with filter_title=title %} By {{ filter_title }} {% endblocktranslate %}</h3>
-<ul>
-{% for choice in choices %}
-    <li{% if choice.selected %} class="selected"{% endif %}>
-    <a href="{{ choice.query_string|iriencode }}">{{ choice.display }}</a></li>
-{% endfor %}
-</ul>
+<details {% if choices|length < 10 %} open {% endif %}>
+    <summary>
+        {% blocktranslate with filter_title=title %} By {{ filter_title }} {% endblocktranslate %}
+    </summary>
```

Comment:
Small edit to generate a cleaner html
```suggestion
<details{% if choices|length < 10 %} open{% endif %}>
    <summary>{% blocktranslate with filter_title=title %} By {{ filter_title }} {% endblocktranslate %}</summary>
```

Reply:
Thanks @pauloxnet!

I'll resolve it as carltongibson comment:
> I don't think I'd worry about auto-collapsing (< 10) here.